mmv1/products/transcoder/Job.yaml

```diff
@@ -75,6 +75,7 @@ examples:
     ignore_read_extra:
       - 'state'
       - 'end_time'
+      - 'start_time'
```

Comment:
should these be alphasorted, or is `state` a special one?

Reply:
we don't have any particular requirement, I'd prefer `start_time` preceding `end_time` purely for aesthetic reasons though

Reply:
Okies. Done.